sfdx-project.json

```diff
@@ -9,10 +9,10 @@
         }
     ],
     "namespace": "",
-    "sourceApiVersion": "48.0",
+    "sourceApiVersion": "49.0",
     "sfdcLoginUrl": "https://login.salesforce.com",
     "packageAliases": {
-        "LWCRecipes": "0HoB0000000GmiJKAS",
+        "lWCRecipes": "0HoB0000000GmiJKAS",
```

Comment:
Why the lower case L?

Reply:
Ah that was from running `force:project:update`. I'll fix that.

Reply:
Updated with proper package name case.